solutions/cses-1076.mdx

```diff
@@ -2,16 +2,16 @@
 id: cses-1076
 source: CSES
 title: Sliding Median
-author: Isaac Noel
+author: Isaac Noel, Arpan Banerjee
 ---
 
 **Time Complexity:** $\mathcal{O}(N\log N)$
 
-## Abstract
+## Abstract 1
```

Comment:
If you're gonna rename this to Abstract 1, you should have an Abstract 2 somewhere.